packages/fuels-accounts/Cargo.toml

```diff
@@ -51,4 +55,5 @@ std = [
   "dep:cynic",
 ]
 test-helpers = []
-kms-signer = ["dep:aws-sdk-kms", "dep:aws-config"]
+accounts-aws-kms-signer = ["dep:aws-sdk-kms", "dep:aws-config"]
+accounts-google-kms-signer = ["dep:google-cloud-kms"]
```

Comment:
I'd remove the `accounts-*` prefix since we are in `accounts`. 

Add it to the fuels package features as a sort of namespacing.